yt/visualization/volume_rendering/scene.py

```diff
@@ -216,13 +226,31 @@ def render(self, camera=None):
         self._last_render = bmp
         return bmp
 
-    def save(self, fname=None, sigma_clip=None):
-        r"""Saves the most recently rendered image of the Scene to disk.
+    def _sanitize_render(self, render=True):
```

Comment:
one last comment maybe: I don't think it's useful or worth it to have a default value here